kubernetes/samples/scripts/create-soa-domain/docs-source/Persisting-SOA-Adapters-Customizations.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Persisting SOA Adapters Customizations"
+date: 2019-12-05T06:46:23-05:00
+weight: 2
+description: "The adapters (such as DB Adapter, File Adapter etc) in a SOA domain can be customized based on users requirement"
+---
+
+The lifetime for any customization done in a file on a server pod is upto the lifetime of that pod, the changes are not persisted once the pod goes down or restarted.
```

Comment:
upto  -> up to
persisted once the pod -> persisted after the pod
or restarted -> or is restarted